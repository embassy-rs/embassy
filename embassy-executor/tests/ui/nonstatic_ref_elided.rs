#![cfg_attr(feature = "nightly", feature(impl_trait_in_assoc_type))]

#[embassy_executor::task]
async fn foo(_x: &u32) {}

fn main() {}
