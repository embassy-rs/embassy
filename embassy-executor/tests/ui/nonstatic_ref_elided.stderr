error: Arguments for tasks must live forever. Try using the `'static` lifetime.
 --> tests/ui/nonstatic_ref_elided.rs:4:18
  |
4 | async fn foo(_x: &u32) {}
  |                  ^
